lighthouse-core/fraggle-rock/gather/timespan-runner.js

```diff
@@ -51,6 +51,7 @@ async function startTimespan(options) {
         {
           url: finalUrl,
           config,
+          computedCache: new Map(),
```

Comment:
Should we be sharing computed artifacts between phases?

Reply:
Yes, but I don't imagine it will really matter in practice as there are no `dependencies` for anything but `getArtifact`